cmd/k8s-operator/deploy/manifests/operator.yaml

```diff
@@ -145,9 +145,17 @@ spec:
                             isExitNode:
                                 description: IsExitNode is set to true if the Connector acts as an exit node.
                                 type: boolean
+                            magicDNSName:
```

Comment:
RE: tailnets that don't have MagicDNS enabled, perhaps call this field 'hostname' and add a comment that 'if MagicDNS is enabled' for you tailnet then this is the MagicDNS name of the node
+ links to 
https://tailscale.com/kb/1098/machine-names
https://tailscale.com/kb/1081/magicdns